Documentation/ProjectFile/gml/surfaces/surface/element/i_element.md

```diff
@@ -1 +1,3 @@
-\ogs_missing_documentation
+Defines a triangle by its three vertices.
+The face normal of the triangle is determined such that the vertices are
+located counter-clockwise around it (right-hand rule).
```

Comment:
As far as I aware there isn't any precondition concerning the point ordering of the triangle. I am even unsure if the ordering have to be consistent within a surface. Maybe @rinkk knows it better?!